tests/test_api.py

```diff
@@ -722,3 +729,49 @@ def test_create_build_config_auto_start(self):
 
         build_response = osbs._create_build_config_and_build(build_request)
         assert build_response.json == {'spam': 'maps'}
+
+    def test_scratch_build_config(self):
```

Comment:
Could we add a test for passing 'scratch=True' to the create_build() API method?